File: trunk/resources/scop_1.5.1/examples/event_listener.cpp
// event_listener.cpp - DMI - 7-9-02

#include <scop.h>

int main()
{
	int sock;
	char *msg;
	
	sock = scop_open("localhost", "event_listener");
	scop_listen(sock, "news");
	while(1)
	{
		msg = scop_get_message(sock);
		printf("Received <%s>\n", msg);
		delete[] msg;
	}
	
	return 0;
}

File: trunk/resources/scop_1.5.1/java/examples/EventListener.java
// EventListener.java - DMI - 21-9-02

import java.io.*;

public class EventListener
{
	public static void main(String[] args)
	{
		SCOP scop;
		String msg;

		scop = new SCOP("localhost", "EventListener");
		scop.listen("news");
		
		while(true)
		{
			msg = scop.get_message();
			System.out.println("Received <" + msg + ">");
		}
	}
}
